src/Geopilot.Frontend/src/pages/admin/organisationDetail.tsx

```diff
@@ -0,0 +1,233 @@
+import { FieldValues, FormProvider, useForm } from "react-hook-form";
+import { CircularProgress, Stack, Typography } from "@mui/material";
+import { FlexBox, FlexRowEndBox, FlexRowSpaceBetweenBox, GeopilotBox } from "../../components/styledComponents.ts";
+import SaveOutlinedIcon from "@mui/icons-material/SaveOutlined";
+import { useCallback, useContext, useEffect, useMemo, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { PromptContext } from "../../components/prompt/promptContext.tsx";
+import CancelOutlinedIcon from "@mui/icons-material/CancelOutlined";
+import { ChevronLeft, UndoOutlined } from "@mui/icons-material";
+import { useNavigate, useParams } from "react-router-dom";
+import { FormAutocomplete, FormContainer, FormInput } from "../../components/form/form.ts";
+import { Mandate, Organisation, User } from "../../api/apiInterfaces.ts";
+import { useApi } from "../../api";
+import { useGeopilotAuth } from "../../auth";
+import { useControlledNavigate } from "../../components/controlledNavigate";
+import { PromptAction } from "../../components/prompt/promptInterfaces.ts";
+import { FormAutocompleteValue } from "../../components/form/formAutocomplete.tsx";
+
+export const OrganisationDetail = () => {
+  const { t } = useTranslation();
+  const { user } = useGeopilotAuth();
+  const formMethods = useForm({ mode: "all" });
+  const { showPrompt } = useContext(PromptContext);
+  const { fetchApi } = useApi();
+  const { registerCheckIsDirty, unregisterCheckIsDirty, checkIsDirty, leaveEditingPage, navigateTo } =
+    useControlledNavigate();
+  const navigate = useNavigate();
+  const { id } = useParams<{
+    id: string;
+  }>();
+
+  const [organisation, setOrganisation] = useState<Organisation>();
+  const [mandates, setMandates] = useState<Mandate[]>();
+  const [users, setUsers] = useState<User[]>();
+
+  useEffect(() => {
+    registerCheckIsDirty(`/admin/organisations/${id}`);
+
+    return () => {
+      unregisterCheckIsDirty(`/admin/organisations/${id}`);
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    if (checkIsDirty) {
+      if (formMethods.formState.isDirty) {
+        const promptActions: PromptAction[] = [
+          { label: "cancel", icon: <CancelOutlinedIcon />, action: () => leaveEditingPage(false) },
+          {
+            label: "reset",
+            icon: <UndoOutlined />,
+            action: () => leaveEditingPage(true),
+          },
+        ];
+        if (formMethods.formState.isValid) {
+          promptActions.push({
+            label: "save",
+            icon: <SaveOutlinedIcon />,
+            variant: "contained",
+            action: () => {
+              saveOrganisation(formMethods.getValues() as Organisation, false).then(() => leaveEditingPage(true));
+            },
+          });
+        }
+        showPrompt("unsavedChanges", promptActions);
+      } else {
+        leaveEditingPage(true);
+      }
```

Comment:
PP: Etwas vereinfachen mit gatecheck style. Evtl lässt sich die Logik mit cancel, reset, save gleich in eine wiederverwendbare Komponente packen.
```suggestion
    if (!checkIsDirty) {
     leaveEditingPage(true);
     return;
    }
      if (formMethods.formState.isDirty) {
        ...
```

Reply:
Ich räume in einem späteren PR auf, sobald alles migriert ist und die Duplikate abschliessend klar sind.